README.md

```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed
+for developers who want a fresh Ubuntu environment with a single command. It uses
+KVM on Linux, Hyper-V on Windows HyperKit on MacOS to run the VM with minimal
```

Comment:
Need `, and` between `Windows` and `HyperKit`.

Reply:
Done.